src/components/notifications/Discord.vue

```diff
@@ -16,4 +16,36 @@
         <label for="discord-prefix-message" class="form-label">{{ $t("Prefix Custom Message") }}</label>
         <input id="discord-prefix-message" v-model="$parent.notification.discordPrefixMessage" type="text" class="form-control" autocomplete="false" :placeholder="$t('Hello @everyone is...')">
     </div>
+
+    <div class="mb-3">
+        <label for="discord-message-type" class="form-label">{{ $t("Select message type") }}</label>
+        <br>
+        <select id="discord-message-type" v-model="$parent.notification.discordChannelType" class="form-select">
+            <option value="normal">{{ $t("Normal Message") }}</option>
+            <option value="createNewForumPost">{{ $t("Create new forum post") }}</option>
+            <option value="postToThread">{{ $t("Post to existing thread") }}</option>
+        </select>
+    </div>
+
+    <div v-if="$parent.notification.discordChannelType === 'createNewForumPost'">
+        <div class="mb-3">
+            <label for="discord-target" class="form-label">{{ $t("Forum post name") }}</label>
+            <input id="discord-target" v-model="$parent.notification.postName" type="text" class="form-control" autocomplete="false" :placeholder="$t('Status Changed')">
+        </div>
+    </div>
+    <div v-if="$parent.notification.discordChannelType === 'postToThread'">
+        <div class="mb-3">
+            <label for="discord-target" class="form-label">{{ $t("Thread ID") }}</label>
```

Comment:
How common is the thread id?
Do you think adding a helptext for how to get this id is nessesary?

Reply:
I'm adding a helptext linking to the discord help page on how to get ids